cmd/src/api.go

```diff
@@ -123,8 +123,25 @@ func gqlURL(endpoint string) string {
 	return endpoint + "/.api/graphql"
 }
 
+// authenticationHeader returns the header name and header value required to authenticate a request,
+// or `"", ""` if one calls this function when the config has no AccessToken information
+func authenticationHeader(cfg *config) (string, string) {
```

Comment:
I did actually consider that, but just because the HTTP header is `Authorization:` doesn't change the fact that the header is providing _authentication_ (unless there is some dark corner of RFC7644 I don't know about)

I'm certainly open to changing it if you feel it's distracting, but that's why it's that way